WordPress/Helpers/ArrayWalkingFunctionsHelper.php

```diff
@@ -0,0 +1,108 @@
+<?php
+/**
+ * WordPress Coding Standard.
+ *
+ * @package WPCS\WordPressCodingStandards
+ * @link    https://github.com/WordPress/WordPress-Coding-Standards
+ * @license https://opensource.org/licenses/MIT MIT
+ */
+
+namespace WordPressCS\WordPress\Helpers;
+
+use PHP_CodeSniffer\Files\File;
+use PHPCSUtils\Utils\PassedParameters;
+
+/**
+ * Helper functions and function lists for checking whether a function is one which applies a callback to an array.
```

Comment:
```suggestion
 * Helper methods and function lists for checking whether a function applies a callback to an array.
```

Reply:
Re: `methods` - I think the same phrasing was also used in the other helpers. We should either fix them all or leave them all as is... I'm tempted to leave it for now as a `method` is a function after all, just one which is within an OO structure.